Example/PrimerSDK/AppViewController.swift

```diff
@@ -17,21 +17,34 @@ class AppViewController: UIViewController, UIPickerViewDataSource, UIPickerViewD
     @IBOutlet weak var countryCodeTextField: UITextField!
```

Comment:
This is the initial controller of the Example app, the one that the user can set env, amount, currency, etc. I also added accessibility ids.